fdbserver/ClusterController.actor.cpp

```diff
@@ -5349,6 +5420,84 @@ ACTOR Future<Void> monitorBlobManager(ClusterControllerData* self) {
 	}
 }
 
+ACTOR Future<Void> startEncryptKeyProxy(ClusterControllerData* self) {
+	TraceEvent("CCEncryptKeyProxy_Start", self->id).log();
+	loop {
+		try {
+			if (self->db.serverInfo->get().encryptKeyProxy.present()) {
+				// Existing encrptKeyProxy registers while waiting, so skip.
+				return Void();
+			}
+
+			// Recruit EncryptKeyProxy in the same datacenter as the ClusterController.
+			// This should always be possible, given EncryptKeyProxy is stateless, we can recruit EncryptKeyProxy
+			// on the same process as the CluserController.
+			std::map<Optional<Standalone<StringRef>>, int> id_used;
+			id_used[self->clusterControllerProcessId]++;
```

Comment:
I think the process count for this pid gets incremented in `getWorkerForRoleInDatacenter`

Reply:
The increment here to mark usage of the ClusterController process; the `getWorkerForRoleInDatacenter` marks the usage for the newly recruited process I believe.